test/integration/targets/include_import/role/test_import_role.yml

```diff
@@ -8,6 +8,7 @@
     test_var: templating test in playbook
     role_vars:
       where_am_i_defined: in the playbook
+    tasks_file_name: canary3
```

Comment:
What is the purpose of defining this var here?

Reply:
That was leftover from an `import_role` task I removed since it's not possible to use host vars with `import_role`. I'll remove it.

Reply:
:+1: shipit!